packages/turf-isolines/index.d.ts

```diff
@@ -1,11 +1,13 @@
-/// <reference types="geojson" />
+import {Points, MultiLineStrings} from '@turf/helpers'
 
-type Points = GeoJSON.FeatureCollection<GeoJSON.Point>;
-type LineStrings = GeoJSON.FeatureCollection<GeoJSON.LineString>;
+interface Properties {
+    perIsoline: Array<Object>,
```

Comment:
Also `propertiesPerIsoline`, seems a bit more complex to use vs. the `propertiesToAllIsolines`. We might want to reverse them.

```javascript
const properties = {foo: 'bar'}
isolines(points, breaks, 'temperature', properties)
isolines(points, breaks, 'temperature', properties, [{name: 'break1'}, {name: 'break2'])
```